packages/components/src/core/Tag/index.stories.tsx

```diff
@@ -269,6 +378,7 @@ function LivePreviewDemo(props: Args): JSX.Element {
         {HOVERABLE_OPTIONS.map((hover) => {
           return (
             <TagState
+              {...props}
```

Comment:
I think this is now needed here so that `...props` are passed to the next loop and it can exist externally and be used by multiple stories (at least it wasn't working at all until I added this line...)